core/routemgmt/common/apigw-utils.js

```diff
@@ -954,16 +950,14 @@ function makeResponseObject(resp, isWebAction) {
     return resp;
   }
 
-  var bodystr;
+  var bodystr = resp;
   if (typeof resp === 'string') {
-    bodystr = makeJsonString(resp);
-  } else {
-    bodystr = JSON.stringify(resp);
+    bodystr = JSON.parse(makeJsonString(resp));
   }
   retobj = {
     statusCode: 200,
     headers: { 'Content-Type': 'application/json' },
-    body: new Buffer(bodystr).toString('base64')
+    body: bodystr
```

Comment:
update comment above to remove the base64 business
* body: Base64 encoded JSON error string